.github/workflows/get-started-dotnet.yml

```
@@ -55,6 +57,24 @@ jobs:
       run: dotnet build --no-restore
       working-directory: get-started/azure-auth/cli/src/
 
+  build-dotnet-dependency-injection:
+
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    needs: []
```

(and replicate to other jobs please so they run in parallels and not sequentially)

---

I changed all the jobs to have the same structure (where the actions will run in parallel).

This is commit 7, you might want to squash this PR